clang/test/lab1/kanakov_roman/test.cpp

```diff
@@ -0,0 +1,107 @@
+// RUN: split-file %s %t
+
```

Comment:
Please, add checks for local variables usage

Reply:
Done